src/services/api/scheduling.ts

```diff
@@ -55,19 +59,57 @@ function generateSchedulingConfiguration({
       error,
     )
     switch (error.status) {
-      case 401: {
```

Comment:
401 case has been removed?

Reply:
They are unathenticated routes

Reply:
`/scheduling/generate-booking-url` will return a 401 if the form requires auth and the user has not logged in and they somehow run this function.

Reply:
ah